eth_tester/main.py

```diff
@@ -505,8 +505,9 @@ def revert_to_snapshot(self, snapshot_id):
         for log_filter in self._log_filters.values():
             self._revert_log_filter(log_filter)
 
-    def reset_to_genesis(self):
-        self.backend.reset_to_genesis()
+    def reset_to_genesis(self, genesis_parameter_overrides=None, genesis_state_overrides=None):
```

Comment:
I'm thinking that we should not expose this API at the `ETHTester` level and only support it at the backend level.  It seems reasonable that some backends might not support custom genesis params and I think it is reasonable to require manual instantiation of the backend when you wish to override genesis parameters.